Python/Product/Profiling/Profiling/ProfiledProcess.cs

```diff
@@ -150,6 +155,38 @@ sealed class ProfiledProcess : IDisposable {
         }
 
         private string GetPerfToolsPath() {
+            using (var baseKey = RegistryKey.OpenBaseKey(RegistryHive.LocalMachine, RegistryView.Registry32)) {
+                if (baseKey == null) {
+                    throw new InvalidOperationException("Cannot open system registry");
+                }
+
+                using (var key = baseKey.OpenSubKey(@"Software\Microsoft\VisualStudio\VSPerf")) {
+                    var path = key?.GetValue("CollectionToolsDir") as string;
+                    if (!string.IsNullOrEmpty(path)) {
+                        if (_arch == ProcessorArchitecture.Amd64) {
+                            path = PathUtils.GetAbsoluteDirectoryPath(path, "x64");
+                        }
+                        if (Directory.Exists(path)) {
+                            return path;
+                        }
+                    }
+                }
+
+                using (var key = baseKey.OpenSubKey(@"SOFTWARE\Microsoft\VisualStudio\RemoteTools\{0}\DiagnosticsHub".FormatUI(AssemblyVersionInfo.VSVersion))) {
```

Comment:
It probably won't make much of a difference because the formatted parameter is a string, but shouldn't registry keys use invariant culture?